src/vm/eventpipe.cpp

```diff
@@ -751,11 +751,6 @@ void EventPipe::WriteEventInternal(EventPipeEvent &event, EventPipeEventPayload
 
     // Get the current thread;
     Thread *pThread = GetThread();
-    if(pThread == NULL)
```

Comment:
Can `pThread` be NULL in the rest of the method now? If yes, do you need to check for pThread==NULL in all places that access it?

Reply:
Yes. That was added now. Thanks